include/cpp/jank/runtime/obj/persistent_vector.hpp

```diff
@@ -5,9 +5,19 @@
 
 namespace jank::runtime
 {
+  namespace obj
+  {
+    using persistent_vector = static_object<object_type::persistent_vector>;
```

Comment:
This will be a problem, since we're instantiating `static_object<object_type::persistent_vector>` before it's defined. It's defined in this file, starting at line 17. That's why this was provided after. I know you need the transient version before, so let's just have both.